services/movie.py

```diff
@@ -0,0 +1,49 @@
+from typing import Optional
+
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: Optional[list[int]] = None,
+        actors_ids: Optional[list[int]] = None
+) -> QuerySet:

```

Comment:
Consider adding type hinting for the return value of the get_movies() function to improve code readability. For example: `-> QuerySet[Movie]:`